src/Pages/ExercisesPage/ExercisesPage.js

```diff
@@ -1,3 +1,27 @@
+<<<<<<< HEAD
```

Comment:
here also it looks like something after resolving conflicts left (<<<<<<<. =======)

Reply:
Fixed. Strange.